docs/HowtoTDE.md

```diff
@@ -0,0 +1,218 @@
+# NuoDB Transparent Data Encryption in Kubernetes
+
+## Introduction
+
+NuoDB offers strong protection for both _data at rest_ and _data in transit_ allowing enterprises to implement and enforce needed company security policies. This helps you to protect your valuable company data. Transport Layer Security (TLS) secures data in transit within NuoDB domain by encrypting all network communication and the data which is in motion. Transparent Data Encryption (TDE) allows data at rest to be encrypted before being written to disk.
+
+Before TDE encryption can be enabled on a database, `NuoDB Admin` must be configured with a _storage password_ for each database.
+`NuoDB Admin` is responsible for propagating storage passwords within the admin layer and to database engines.
+All SMs and TEs in the database will use this storage password as a key to secure the encrypted data.
```

Comment:
It might be confusing to people who are familiar with NuoDB that TEs need the password. Can you include a note about TEs using it to encrypt data that is spilled to disk from memory?

Reply:
I've added some explanation of what is encrypted.